docs/design/keyboard-shortcuts.md

```diff
@@ -1,314 +1,473 @@
+1. [Map custom actions to their functions](#map-custom-actions-to-their-functions) using the [Office.actions.associate](/javascript/api/office/office.actions#office-office-actions-associate-member) API.
+
+## Prerequisites
+
+Keyboard shortcuts only work on platforms that support the following requirement sets. For information about requirement sets and how to work with them, see [Specify Office applications and API requirements](../develop/specify-office-hosts-and-api-requirements.md).
+
+- [SharedRuntime 1.1](/javascript/api/requirement-sets/common/shared-runtime-requirement-sets)
+- [KeyboardShortcuts 1.1](/javascript/api/requirement-sets/common/keyboard-shortcuts-requirement-sets) (required if the add-in provides its users with the option to customize keyboard shortcuts)
+
+Keyboard shortcuts are currently only supported in **Excel** and only in these platforms and builds:
+
+- Excel on the web
+- Excel on Windows: Version 2102 (Build 13801.20632) and later
+- Excel on Mac: Version 16.48 and later
+
+## Configure the manifest to use a shared runtime
+
+To customize keyboard shortcuts for your add-in, you must first configure the add-in manifest to use a [shared runtime](../testing/runtimes.md#shared-runtime). Select the tab for the type of manifest you're using.
+
+# [Unified app manifest for Microsoft 365](#tab/jsonmanifest)
 
 > [!NOTE]
-> To start with a working version of an add-in with keyboard shortcuts already enabled, clone and run the sample [Excel Keyboard Shortcuts](https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/excel-keyboard-shortcuts). When you're ready to add keyboard shortcuts to your own add-in, continue with this article.
+> Implementing keyboard shortcuts with the unified app manifest for Microsoft 365 is in public developer preview. It's currently only available in **Excel**. This shouldn't be used in production add-ins. We invite you to try it out in test or development environments. For more information, see the [Public developer preview app manifest schema](/microsoftteams/platform/resources/schema/manifest-schema-dev-preview).
 
-There are three steps to add keyboard shortcuts to an add-in.
+1. In your add-in project, open the **manifest.json** file.
+1. Add the following object to the "extensions.runtimes" array. Note the following about this markup.
 
-1. [Configure the add-in's manifest](#configure-the-manifest).
-1. [Create or edit the shortcuts JSON file](#create-or-edit-the-shortcuts-json-file) to define actions and their keyboard shortcuts.
-1. [Add one or more runtime calls](#create-a-mapping-of-actions-to-their-functions) of the [Office.actions.associate](/javascript/api/office/office.actions#office-office-actions-associate-member) API to map a function to each action.
+    - The SharedRuntime 1.1 requirement set is specified in the "requirements.capabilities" object. This is required to support custom keyboard shortcuts.
```

Comment:
We should delete this bullet. I'll send email. Too complicated to explain in a comment.

Reply:
Deleted.